client/src/main/java/lighthouse/subwindows/UpdateFXWindow.java

```diff
@@ -73,10 +75,14 @@ protected void updateItem(@Nullable UFXProtocol.Update item, boolean empty) {
             else
                 descriptionLabel.setText("");
         });
+        
+        // Load localized strings
+        closeButton.setText(_("Close"));
+        pinBtn.setText(_("Always use this version"));
```

Comment:
One thing I just realised is that whilst the UpdateFX window itself may be translated, the update descriptions will not be. I'm not sure I want to wait for translations each time I push an update, though as the descriptions are often quite repetitive perhaps we can just have a few pre-canned vague descriptions that are used as the translation.

At any rate, I'll put i18n on my UpdateFX to do list for when I next revisit it (soon).